src/angular-app/languageforge/lexicon/settings/project-settings.component.html

```diff
@@ -27,7 +27,7 @@
                     </div>
                     <div class="form-group row">
                         <label class="col-form-label col-12">Project Owner</label>
-                        <div class="controls col-12 notranslate">
+                        <div id="e2e-test-project-owner" class="controls col-12 notranslate">
```

Comment:
The [Playwright docs](https://playwright.dev/docs/1.19/selectors#define-explicit-contract) suggest using the `data-test-id` attribute for things like this. This is fine here, because this `id` attribute won't be duplicated, but in anything that appears more than once on a page, the `id` attribute is supposed to be unique. So it's good practice to use the `data-test-id` attribute when you need to mark a specific element for your E2E tests.

Reply:
Yes, good to know. Thanks for pointing this out!